docs/migration/manual-migration.mdx

```diff
@@ -11,36 +11,71 @@ This manual migration process should be run after the
 [automated migration process](./automated-migration), to complete the missing
 parts, or debug issues in the migration CLI output.
 
+## Components
+
+### `HomeSeo`
+
+The `HomeSeo` component has been removed in favor of
+[Remix Meta](https://remix.run/docs/en/main/route/meta-v2)
+
+We have introduced a new Meta generation helper `generateMetas`.
```

Comment:
**issue:** `generateMeta` documentation isn't relevant here. This section must stay to the point with actual actions integrators must tackle to cope with our decisions.

**suggestion:**
- replacing `*Seo` components with Remix meta is a more generic concern. Not only related to HomeSeo. I think the section should be labeled after this change instead of focusing on `HomeSeo`
- move `generateMeta` documentation to https://deploy-preview-724--heuristic-almeida-1a1f35.netlify.app/docs/remixed/api-reference/front-commerce-remix/
- explain clearly what **integrators must do** if they have errors related to these components, and if they have override
  - remove imports and usage of `HomeSeo`
  - customize xxxxxx to maintain their custom SEO content

Reply:
Update done about SEO related documentation.
About Conceptual changes I don't really now how to start and introduce this

Reply:
Ok, no worries. We'll add this new section later.